cloudsim-plus-testbeds/src/main/java/org/cloudsimplus/migration/ExamplePowerVmAllocationPolicyMigrationStaticThreshold.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
```

Comment:
You can't never, never remove the original copyright notice provided by CloudSim
neither the authors tag from the JavaDoc. If you do that you are violating the license terms.
If you changed an existing file, what you are allowed to do is to include your name at the end of the list of authors.